Example/AppDelegate.swift

```diff
@@ -23,12 +23,13 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 	}
 
 	private func configureManualView() {
-		manualCircularProgress.onCancelled = {
-			print("Manual cancelled!")
+		manualCircularProgress.onCancelled = { circularProgress in
+			circularProgress.alphaValue = 0.3
```

Comment:
Why not just `manualCircularProgress.alphaValue = 0.3`? I usually pass on self in closures too, but is it really that beneficial?

Reply:
I was juggling a bunch of circular progresses in the example app at some point. It cleaned up the code a bit then. I think it's better to now have it moving forward.